### PR TITLE
Gough 11-12

### PR DESCRIPTION
**Pull request by maijapapaya**
@jaredh159 @Henderjay

### CODE REVIEW

File: history-of-the-quakers-v4/original/17-book-7-chapter-11.adoc [hunk 1]
```diff
@@ -198,11 +203,15 @@ but rather pointed to bring them under difficulties and sufferings.
...
+--
+
+[.letter-heading]
 To Thomas Penn and Richard Penn, proprietaries of Pennsylvania,
 and the three lower counties of New Castle, Kent and Sussex upon Delaware.
 
-The Address of the People called Quakers,
-on behalf of their Brethren inhabiting the said lower Counties, respectively showeth,
+[.blurb]
+=== The Address of the People called Quakers, on behalf of their Brethren inhabiting the said lower Counties, respectively showeth,
```
Comment (maijapapaya) [inline history-of-the-quakers-v4/original/17-book-7-chapter-11.adoc:214]: Not sure if this should be a blurb.

File: history-of-the-quakers-v4/original/17-book-7-chapter-11.adoc [hunk 2]
```diff
@@ -251,7 +260,7 @@ shall be in any case molested or prejudiced in his or their person or estate,
 because of his or their conscientious persuasion or practice,
 nor be compelled to frequent or maintain any religious worship, place,
 or ministry contrary to his or their minds,
-or to do or to suffer any other act or thing contrary to their religious persuasion.`"
+or to do or to suffer any other act or thing contrary to their religious persuasion.
 
 May it please the Proprietaries,
```
Comment (maijapapaya) [inline history-of-the-quakers-v4/original/17-book-7-chapter-11.adoc:265]: Not sure what to do with this in the middle of the embed.

File: history-of-the-quakers-v4/original/18-book-7-chapter-12.adoc [hunk 3]
```diff
@@ -285,6 +289,10 @@ nominated twelve friends as representatives of the yearly meeting,
 and directed the respective Quarterly meetings to
 appoint four members each to represent them.
 
+[.embedded-content-document.letter]
+--
+
+[.letter-heading]
 The Services to be performed by the Meeting for Sufferings, viz.
```
Comment (maijapapaya) [inline history-of-the-quakers-v4/original/18-book-7-chapter-12.adoc:296]: Wasn't sure about this.

File: history-of-the-quakers-v4/original/17-book-7-chapter-11.adoc (hunk 2 above)
Comment (jaredh159) [reply history-of-the-quakers-v4/original/17-book-7-chapter-11.adoc:265]: Yeah, I think let's just leave it alone 👍

File: history-of-the-quakers-v4/original/18-book-7-chapter-12.adoc (hunk 3 above)
Comment (jaredh159) [reply history-of-the-quakers-v4/original/18-book-7-chapter-12.adoc:296]: Yeah, I don't think this is a letter.  I sort of lean towards leaving off a type, and then changing line 296 to just be a plain sub-heading, not a letter heading.

**Comment by jaredh159** [comment]
looks great, thanks! 👍

**Pull request merged**